packages/components/src/Form/Inputs/Radio/Radio.tsx

```diff
@@ -28,21 +28,29 @@ import React, { forwardRef, Ref } from 'react'
 import styled from 'styled-components'
 import { reset, space, SpaceProps } from '@looker/design-tokens'
 import { InputProps, pickInputProps } from '../InputProps'
+import { ValidationType } from '../../ValidationMessage'
+import { inputTextValidation } from '../InputText'
 import { FauxRadio } from './FauxRadio'
 
 export interface RadioProps
   extends SpaceProps,
     Omit<InputProps, 'readonly' | 'type' | 'checked' | 'onClick'> {
   checked?: boolean
+  validationType?: ValidationType
 }
 
 const RadioLayout = forwardRef(
   (props: RadioProps, ref: Ref<HTMLInputElement>) => {
-    const { className, ...restProps } = props
+    const { className, validationType, ...restProps } = props
 
     return (
       <div className={className}>
-        <input type="radio" {...pickInputProps(restProps)} ref={ref} />
+        <input
+          type="radio"
+          aria-invalid={validationType === 'error' ? 'true' : undefined}
+          {...pickInputProps(restProps)}
+          ref={ref}
```

Comment:
Since we don't want the spread `...` to overwrite any of the key settings on the radio input it should be done before those attributes get assigned explicitly.

```suggestion
          {...pickInputProps(restProps)}
          type="radio"
          aria-invalid={validationType === 'error' ? 'true' : undefined}
          ref={ref}
```